plugins/newegg.py

```diff
@@ -70,9 +70,9 @@ def format_item(item, show_url=True):
         url = web.try_shorten(ITEM_URL.format(item["NeweggItemNumber"]))
         return "\x02{}\x02 ({}) - {} - {} - {}".format(title, price, rating,
                                                        tag_text, url)
-    else:
-        return "\x02{}\x02 ({}) - {} - {}".format(title, price, rating,
-                                                  tag_text)
+
+    return "\x02{}\x02 ({}) - {} - {}".format(title, price, rating,
```

Comment:
This should fit on one line

Reply:
I'm limiting the changes to what I set out to do, specifically just removing the `return-else` statements